src/jvm/main/org/jetbrains/kotlinx/lincheck/transformation/TraceDebuggerAgent.kt

```diff
@@ -25,20 +25,14 @@ import java.security.ProtectionDomain
  * in order to enable trace debugging plugin functionality.
  */
 internal object TraceDebuggerAgent {
-    val classUnderTraceDebugging: String = System.getProperty("traceDebugger.className", "")
-    val methodUnderTraceDebugging: String = System.getProperty("traceDebugger.methodName", "")
```

Comment:
If class and method names are now passed as jvm agent arguments instead of system properties, should it also be addressed on the plugin side?

Reply:
Yes. I did this before `develop` branch become default for the trace debugger plugin. I will prepare an MR for the plugin.